ArtOfIllusion/src/artofillusion/ViewerCanvas.java

```diff
@@ -1436,7 +1436,7 @@ public void centerToPoint(Point pointOnView)
     Vec3 cp = pointInSpace.plus(cz.times(-distToPlane));
     coords.setOrigin(cp);
 
-    animation.start(coords, pointInSpace, scale, orientation, navigation);
+    if(null != animation) animation.start(coords, pointInSpace, scale, orientation, navigation);
```

Comment:
Condition order is backwards here. For readability, it should be `if(animation != null)`